src/main/scala/org/broadinstitute/dsde/firecloud/model/Workspace.scala

```diff
@@ -29,11 +29,20 @@ case class EntityId(entityType: String, entityName: String)
 case class BagitImportRequest(bagitURL: String, format: String)
 
 case class PfbImportRequest(
-  url: String,
+  url: Option[String],
```

Comment:
making this an option allows us to be lenient during json deserialization of end-user request bodies, and delegate validation to import service.